pyproject.toml

```diff
@@ -6,7 +6,7 @@ authors = [
     { name = "DetachHead", email = "detachhead@users.noreply.github.com" },
 ]
 dependencies = [
-    "pytest<8,>=7",
+    "pytest<9,>=7",
```

Comment:
Do we want to see tests for pytest 7?

Reply:
not worth it imo since barely anything this plugin uses changed between veresion 7 and 8